kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_componentwise.h

```diff
@@ -128,16 +128,35 @@ class ResidualBasedEliminationBuilderAndSolverComponentwise
 
     typedef typename BaseType::ElementsContainerType ElementsContainerType;
 
-    /*@} */
-    /**@name Life Cycle
-    */
-    /*@{ */
+    ///@}
+    ///@name Life Cycle
+    ///@{
 
-    /** Constructor.
-    */
-    ResidualBasedEliminationBuilderAndSolverComponentwise(
+    /**
+     * @brief Default constructor. (with parameters)
+     */
+    explicit ResidualBasedEliminationBuilderAndSolverComponentwise(
+        typename TLinearSolver::Pointer pNewLinearSystemSolver,
+        Parameters ThisParameters
+        ) : ResidualBasedEliminationBuilderAndSolverType(pNewLinearSystemSolver)
+    {
+        // Validate default parameters
+        Parameters default_parameters = Parameters(R"(
+        {
+            "components_wise_variable" : "DISPLACEMENT"
```

Comment:
DISPLACEMENT is incorrect as a componentwise variable...

it should be either a scalar variable (PRESSURE) or a component DISPLACEMENT_X

in any case i would not put a default but rather ask by default for something like "SCALAR_VARIABLE_OR_COMPONENT"

Reply:
OK